HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -4489,21 +4511,19 @@ def self.calc_ceer_from_eer(cooling_system)
   def self.set_fan_power_rated(hvac_system, use_eer_cop)
     hvac_ap = hvac_system.additional_properties
 
+    # Based on RESNET DX Modeling Appendix
+    psc_ducted_watts_per_cfm = 0.414 # W/cfm, PSC fan
+    psc_ductless_watts_per_cfm = 0.414 # W/cfm, PSC fan
```

Comment:
So far as I can tell, psc_ductless_watts_per_cfm is not used for any case. Ductless minisplit is always var speed and defaulted to bpm fans. Room AC and PTHPs are outside the scope.